go.mod

```diff
@@ -58,3 +58,5 @@ require (
 	google.golang.org/grpc v1.48.0 // indirect
```

Comment:
This is unrelated to this file. The question is where is the change log? :)

Reply:
Unbelievably it's not there. I thought I added it but it was in the govcd PR. I'll add it.

Thanks for noticing!

Reply:
Fixed, thanks!